draft-ietf-core-new-block.xml

```diff
@@ -859,9 +837,14 @@ missing-block-number = uint
     </section>
 
     <section anchor="cc" title="Congestion Control">
```

Comment:
```suggestion
    <section anchor="cc" title="Congestion Control for Unreliable Transports">
```

Reply:
Same comment about the usage of Unreliable as you can argue CoAP is a reliable transport when using UDP when using CON.

Reply:
Same as above. We do have reliable and unreliable transport for CoAP. I don't think there is a confusion out there.

Reply:
OK - Text changed.